.github/workflows/build.yml

```diff
@@ -1,9 +1,11 @@
 name: build
 
 on:
-  - workflow_dispatch
-  - push
-  - workflow_call
+  push:
+    branches:
+      - '**'
```

Comment:
Does this need an explicit `tags-ignore`?

Reply:
Seems like it wouldn't hurt to explicitly exclude all tags, and it seems safer to me to do so, in-case we change the branches to something other than "all branches".